core/util.go

```diff
@@ -21,6 +21,7 @@ import (
 	"encoding/json"
 	"encoding/pem"
 	"errors"
+	"expvar"
```

Comment:
So, packages using core will get /debug/vars by virtue of this import.

Maybe we should be explicit about importing expvar in cmd where we set up the other debug handlers?

It'd be a `_ "expvar"` import there.